bibliopixel/main/main.py

```diff
@@ -3,7 +3,7 @@
 
 __all__ = ['main']
 
-COMMANDS = 'demo',  # 'device', 'network', 'run', 'settings'
+COMMANDS = 'demo', 'run',  # 'device', 'network', 'settings'
```

Comment:
I realize this is valid python3 but i really hate this syntax. () around it would look cleaner in my opinion.